user-guide/enterprise-auth/intro.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Using Enterprise Authentication
+description: Learn how to connect to your enterprise authentication provider, and how to leverage data from that provider in your apps.
+position: 1
+publish: true
+slug: enterprise-auth
+---
+
+# Using Enterprise Authentication
+
+NativeScript Sidekick lets you connect to existing enterprise identity and single sign-on solutions. This powerful functionality allows application developers to leverage enterprise user accounts, and to do so while avoiding the complexity of integrating these protocols into mobile apps.
+
+Let’s look at how it works.
+
+* [Step 1. Start you app](#step-1)
+* [Step 2. Select a protocol](#step-2)
+* [Step 3: Complete the configuration](#step-3)
+* [Step 4: Run your app](#step-4)
+
+<h2 id="step-1">Step 1. Start your app</h2>
+
+As a first step, [create a new app]({% slug create-app-from-template %}) using the Enterprise Auth template.
+
+![](/images/enterprise-auth/sidekick-templates.png)
+
+Give your app a name, and click **Create**. Sidekick will take a minute to scaffold your app, and then you’ll be ready to set up the connection to your enterprise auth provider.
+
+<h2 id="step-2">Step 2. Select a protocol</h2>
+
+> **WARNING**: Completing this authentication setup can be hard, as the information you need to obtain varies depending on your auth provider. Therefore, you might need to work with your IT admins to help get the information that you need to make the connection, especially if you’re not familiar with the infrastructure behind your auth provider.
+
+On the Enterprise Auth screen the first thing you’ll see is a collection of radio buttons, asking you to choose between **SAML-Redirect**, **OpenID Connect**, and **OAuth 2**.
+
+![](/images/enterprise-auth/selecting-authentication-service.png)
+
+Knowing which service you need means knowing a little bit about your enterprise authentication provider. Some providers only support one of the above protocols, while others allow you to choose between the protocol you prefer.
+
+Once you’ve figured out which protocol your service supports, select the appropriate radio button and move on to the next step.
+
+<h2 id="step-3">Step 3: Complete the configuration</h2>
+
+Each authentication service requires a different set of values needed to make the connection. Therefore, refer to one of the following guides to help complete the rest of the necessary configuration.
+
+* [Configuring **SAML-Redirect** Authentication]({% slug saml%})
```

Comment:
I have changed the markdown links with slug ones. As mentioned above, this will make it easier to change the docs structure later of, if needed.